crypto/tbs/src/lib.rs

```diff
@@ -264,5 +267,14 @@ mod tests {
         let bsig = combine_valid_shares(shuffle_sigs, threshold);
         let sig = unblind_signature(bkey, bsig);
         assert!(verify(msg, sig, pk));
+
+        // Not enough signatures causes panic
+        for _ in 0..6 {
+            sigs.pop();
+        }
+        let result = std::panic::catch_unwind(|| {
+            combine_valid_shares(sigs.clone().into_iter(), threshold);
+        });
```

Comment:
Would it be very annoying to run this in a separate fn with [`should_panic`](https://doc.rust-lang.org/rust-by-example/testing/unit_testing.html#testing-panics)? Seems cleaner. Otherwise let's go with this.

Reply:
Updated